scripts/near-shell-check.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * This file is a stop-gap for upgrading projects when near-shell improves/modifies logic
```

Comment:
Given that these are throwaway accounts, we don’t have to migrate anything IMO.

Reply:
See replies below